src/components/strategies/useWarning.tsx

```diff
@@ -38,6 +39,8 @@ export const useStrategyWarning = ({
       order: order1,
       buy: false,
     });
+  const marketPrice = useMarketPrice({ base, quote });
```

Comment:
Why not using `useUserMarketPrice` here ?

Reply:
Oversight - just replaced